src/main/java/org/ibp/api/java/impl/middleware/location/LocationServiceImpl.java

```diff
@@ -42,12 +48,14 @@ public long countLocations(final String crop, final LocationSearchRequest locati
 	@Override
 	public List<LocationDTO> searchLocations(final String crop, final LocationSearchRequest locationSearchRequest,
 			final Pageable pageable, final String programUUID) {
-		return this.locationMiddlewareService.searchLocations(locationSearchRequest,	pageable, programUUID);
+		return this.locationMiddlewareService.searchLocations(locationSearchRequest, pageable, programUUID);
 	}
 
 	@Override
 	public void deleteLocation(final Integer locationId) {
 		this.locationValidator.validateCanBeDeleted(locationId);
+		final Set<Integer> entityIds = ImmutableSet.of(locationId);
```

Comment:
@cuenyad  I'd rather moving the program favorites deletion to locationMiddlewareService

Reply:
done.